### PR TITLE
Package component library via Rollup

### PR DESCRIPTION
**Pull request by ohaibbq**
## Description of the change

This PR adds Rollup, a module bundler, to package our component library. It also exposes a new interface for our module's entrypoint `index.js`.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,14 @@
...
   },
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "react-scripts start && rollup -c -w",
```
Comment (nbhargava) [inline package.json:18]: `yarn start` just gives me a blank page. Is it supposed to actively do anything? If not, can we just remove it?

Similarly, that would let us kill `startWithStorybook` and maybe drop `concurrently` as a dependency?

Comment (ohaibbq) [reply package.json:18]: :+1: Let's move this repo to storybook-only!

**Comment by macfarlandian** [comment]
LGTM pending nikhil's comment, commands run great now locally otherwise 🚀

**Review by nbhargava — APPROVED** [review]
Into it

File: package.json [hunk 2]
```diff
@@ -9,36 +9,55 @@
...
   },
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "start-storybook -p 60006 -s public",
```
Comment (macfarlandian) [inline package.json:18]: nit: our convention has been to use `dev` for the main local development entry point

**Pull request merged**